tests/clients/test_job.py

```diff
@@ -709,3 +709,14 @@ def test_job_client_get_subrun(mocker, run_identifier, subrun_identifier):
         ),
         schema_mock.return_value,
     )
+
+
+def test_job_client_cancel_run(mocker, run_identifier=RUN_ID):
```

Comment:
I would just embed the `RUN_ID` in the code below rather than adding this kwarg. kwargs as arguments to test functions are normally used only for providing fixtures (e.g. `mocker`) or values for parameterized tests.

Reply:
Yeah, this is a placeholder as expecting to update Steve to take both number and id, and then it will be rewritten. (draft state)